failing test testCookieOptions

## problem
Just ran the swift-web tests and they fail.

```
Test Case '-[HttpPipelineTests.HttpPipelineTests testCookieOptions]' started.
/private/tmp/.nio-release-tools_bcOmLA/swift-web/Tests/HttpPipelineTests/HttpPipelineTests.swift:79: error: -[HttpPipelineTests.HttpPipelineTests testCookieOptions] : failed - Diff: …

@@ −11,9 +11,9 @@
   Content-Length: 19
   Content-Type: text/html; charset=utf-8
   Set-Cookie: foo=bar; Domain=www.pointfree.co
   Set-Cookie: foo=bar; Domain=www.pointfree.co; HttpOnly; Secure
−  Set-Cookie: foo=bar; Expires=Fri, 13 Feb 2009 23:31:30 GMT
+  Set-Cookie: foo=bar; Expires=Fri, 13 Feb 2009 11:31:30 pm GMT
   Set-Cookie: foo=bar; HttpOnly
   Set-Cookie: foo=bar; Max-Age=3600
   Set-Cookie: foo=bar; Path=/path/to/some/where
   Set-Cookie: foo=bar; SameSite=Lax
```

## guess about what's causing this

It might be a US vs UK locale issue. This is my locale

```
$ locale
LANG="en_GB.UTF-8"
LC_COLLATE="en_GB.UTF-8"
LC_CTYPE="en_GB.UTF-8"
LC_MESSAGES="en_GB.UTF-8"
LC_MONETARY="en_GB.UTF-8"
LC_NUMERIC="en_GB.UTF-8"
LC_TIME="en_GB.UTF-8"
LC_ALL=
```

## environment

- swift-web commit: https://github.com/pointfreeco/swift-web/commit/702ddc3f14adfeaa98befae5556fda97b45425aa
- swift version: 4.2 pre-release
- platform: macOS

--- a/Sources/HttpPipeline/Response.swift
+++ b/Sources/HttpPipeline/Response.swift
@@ -142,4 +142,5 @@ public struct Response {
 
 private let expiresDateFormatter = DateFormatter()
   |> \.dateFormat .~ "EEE, d MMM yyyy HH:mm:ss zzz"
+  |> \.locale .~ Locale(identifier: "en_US_POSIX")
   |> \.timeZone .~ TimeZone(abbreviation: "UTC")

